valkeycompat/adapter.go

```
@@ -729,6 +733,34 @@ func (c *Compat) ClientGetName(ctx context.Context) *StringCmd {
+
+	data, err := resp.AsBytes()
+	if err != nil {
+		cmd.SetErr(err)
+		return cmd
+	}
+
+	
+	n := copy(buf, data)
+	cmd.SetVal(n)
+	if valkey.IsValkeyNil(err) {
+		err = nil
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)